packages/open_aea/connections/p2p_libp2p/connection.yaml

```diff
@@ -71,14 +71,14 @@ config:
   storage_path: null
 cert_requests:
 - identifier: acn
-  ledger_id: fetchai
+  ledger_id: cosmos
```

Comment:
I am not 100% but I think this should be "ethereum" (i.e. the skill side key type). But please confirm. This and related code might help (https://github.com/valory-xyz/open-aea/blob/main/aea/helpers/acn/agent_record.py#L62).